moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -201,13 +201,10 @@ export default function TheMap({
             feature?.properties?.[sourceFeatureId] !== featureUniqueId // From CTN layers
         );
 
-        const isAtLeastOneFeatureRemaining = filteredFeatures.length > 0;
-        return isAtLeastOneFeatureRemaining
-          ? {
-              ...currentComponent,
-              [tableToInsert]: filteredFeatures,
-            }
-          : currentComponent;
```

Comment:
This was a temporary fix that prevented removing the last remaining feature from a component.